### PR TITLE
fix: hide dropdown toolbar on resize

### PR DESCRIPTION
**Pull request by kyuwoo-choi**
### Please check if the PR fulfills these requirements
- [x] It's the right issue type on the title
- [x] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [x] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes/features)
- [ ] Docs have been added/updated (for bug fixes/features)
- [x] It does not introduce a breaking change or has a description of the breaking change

### Description
hide dropdown toolbar on editor resize.
fix wrong occasional dropdown toolbar width

---
Thank you for your contribution to TOAST UI product. 🎉 😘 ✨

### CODE REVIEW

File: src/js/ui/popupDropdownToolbar.js [hunk 1]
```diff
@@ -152,8 +152,16 @@ class PopupDropdownToolbar extends LayerPopup {
     this._eventManager.listen('focus', () => this.hide());
     this._eventManager.listen('closeAllPopup', () => this.hide());
     this._eventManager.listen(PopupDropdownToolbar.OPEN_EVENT, () => {
+      const isShown = this.isShow();
```
Comment (seonim-ryu) [inline src/js/ui/popupDropdownToolbar.js]: isShown 변수에 담을 필요가 있을까 싶어요ㅎㅎ
한 번밖에 사용을 안하기 때문에 조건문에서 바로 처리해줘도 좋을 것 같습니다~

**Comment by seonim-ryu** [comment]
[4/17] 리뷰 완료하였습니다~

**Comment by jinwoo-kim-nhn** [comment]
리뷰 완료하였습니다. 수고하셨습니다.

File: src/js/ui/popupDropdownToolbar.js (hunk 1 above)
Comment (kyuwoo-choi) [reply src/js/ui/popupDropdownToolbar.js]: 그 다음 곧바로 모든 팝업을 닫게되어서 어쩔 수 없네요.

**Pull request merged**